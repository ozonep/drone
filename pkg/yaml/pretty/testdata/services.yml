kind: pipeline
name: default

steps:
- name: test
  image: golang
  commands:
    - go build
    - go test -v

services:
- name: redis
  image: redis:localhost
  ports:
  - 6379
  entrypoint:
  - /bin/redis-server
  command:
  - --port
  - 6380
- name: mysql
  image: mysql:latest
  ports:
  - 3306
  environment:
    MYSQL_USERNAME: root
    MYSQL_PASSWORD:
      from_secret: password

depends_on:
  - foo
  - bar
