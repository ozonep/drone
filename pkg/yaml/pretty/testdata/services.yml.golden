---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: test
  image: golang
  commands:
  - go build
  - go test -v

services:
- name: redis
  image: redis:localhost
  entrypoint:
  - /bin/redis-server
  command:
  - --port
  - 6380
  ports:
  - 6379

- name: mysql
  image: mysql:latest
  environment:
    MYSQL_PASSWORD:
      from_secret: password
    MYSQL_USERNAME: root
  ports:
  - 3306

depends_on:
- foo
- bar

...
